mlos_bench/mlos_bench/run.py

```diff
@@ -137,7 +139,8 @@ def _run(env: Environment, opt: Optimizer,
 
     if not env.setup(trial.tunables, trial.config(global_config)):
         _LOG.warning("Setup failed: %s :: %s", env, trial.tunables)
-        trial.update(Status.FAILED)
+        # FIXME: Use the actual timestamp from the environment.
+        trial.update(Status.FAILED, datetime.now())
```

Comment:
```suggestion
        trial.update(Status.FAILED, datetime.utcnow())
```

To make things more easily comparable, let's standardize on using UTC time everywhere.

Reply:
Actually, this is a temporary hack - I think we should actually use a timestamp from the remote VM. That is, record the time when the benchmark had failed rather than when we've noticed it

Reply:
on a second thought, let's use UTC time everywhere. later we can get timestamps from a remote VM, but it still going to be UTC